Aquarius/Src/Aquarius/Core/Application.h

```diff
@@ -1,11 +1,11 @@
 #pragma once
 
+#include "Aquarius/Core/Layer.h"
 #include "Aquarius/Core/Utility.h"
 #include "Aquarius/Core/Window.h"
 
 #include <memory>
 
```

Comment:
Add newline here so that there are two empty lines between last include and namespace.